### ISSUE TITLE
Broken code snippets on tutorial

### ISSUE THREAD

**Issue by gris** [gatsbyjs/gatsby#26318]
## Summary

What problem(s) did you run into that caused you to request additional documentation? What questions do you think we should answer? What, if any, existing documentation relates to this proposal?

I was working through the [tutorial](https://www.gatsbyjs.org/tutorial/part-one/#familiarizing-with-gatsby-pages) and found that the code snippet below has the ` {/* highlight-start */}`, ` {/* highlight-end */}` lines. The first one breaks the code snippet if you paste it directly. Not a big issue at all, just reporting here to know if these statements are being used, and if not I'm open to create a PR :)
![image](https://user-images.githubusercontent.com/11340665/89739755-aac1e600-da59-11ea-8182-9872c15f9e0f.png)

```jsx
import React from "react"

export default function Home() {
  return (
    {/* highlight-start */}
    <div style={{ color: `purple` }}>
      <h1>Hello Gatsby!</h1>
      <p>What a world.</p>
    {/* highlight-end */}
    </div>
  );
}

```
### Open a pull request

- [ ] Open a pull request with your work including the words "closes #[this issue's number]" in the pull request description

**Comment by mcgallag** [comment]
Ran into the same issue with all the code snippets in the tutorial documentation. Expected behavior is that the "highlighted" lines will be actually highlighted in the documentation, showing the reader what needs to be modified in the tutorial code. Makes it hard to follow, I'm guessing this wasn't intentional.

**Comment by gris** [comment]
It seems that it was fixed after the website remodeling. Cool move 👏
